regexes/oss.yml

```diff
@@ -5,6 +5,20 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+##########
+# Oracle Linux (https://www.oracle.com/linux/)
+##########
+- regex: '.+.el(\d+(?:[_\.]\d+)*)uek'
```

Comment:
we need a more strict rule. what about?
`Linux/(?:\d+\.[\d.-]+)\.el(\d+(?:[_.]\d+)*)uek` 

https://regex101.com/r/xs0M2a/1